tests/test_lint.py

```diff
@@ -419,29 +415,29 @@ def test_check_condition_df(minimal_sbml_model):
 
     # parameter missing in model
     with pytest.raises(AssertionError):
-        lint.check_condition_df(condition_df, sbml_model)
+        lint.check_condition_df(condition_df, ss_model.model)
 
     # fix:
-    sbml_model.createParameter().setId('p1')
-    lint.check_condition_df(condition_df, sbml_model)
+    ss_model.addParameter('p1', 1.0)
+    lint.check_condition_df(condition_df, ss_model.model)
 
     # species missing in model
     condition_df['s1'] = [3.0]
     with pytest.raises(AssertionError):
-        lint.check_condition_df(condition_df, sbml_model)
+        lint.check_condition_df(condition_df, ss_model.model)
 
     # fix:
-    sbml_model.createSpecies().setId('s1')
-    lint.check_condition_df(condition_df, sbml_model)
+    ss_model.addSpecies("[s1]", 1.0)
+    lint.check_condition_df(condition_df, ss_model.model)
 
     # compartment missing in model
-    condition_df['c1'] = [4.0]
+    condition_df['c2'] = [4.0]
```

Comment:
Compartment name changed? Is it that the default compartment in a `simplesbml.SbmlModel` is already `'c1'`?